core/services/vrf/vrf.go

```diff
@@ -119,6 +119,11 @@ var Generator = secp256k1Curve.Point().Base()
 // HashUint256s returns a uint256 representing the hash of the concatenated byte
 // representations of the inputs
 func HashUint256s(xs ...*big.Int) (*big.Int, error) {
+	eulersCriterionPower := div(sub(fieldSize, one), two)
+	_ = eulersCriterionPower
+	Generator := secp256k1Curve.Point().Base()
+	_ = Generator
+	fmt.Println("%d", "foo")
```

Comment:
Looks like a debugging statement?

Reply:
That's there to trigger the CI failures. I'll remove the final commit with these probes, before merging.

Reply:
I've removed them.